tests/foreman/api/test_contentview.py

```diff
@@ -507,6 +523,53 @@ def test_negative_create_with_invalid_name(self):
                 with self.assertRaises(HTTPError):
                     entities.ContentView(name=name).create()
 
+    @tier2
+    def test_composite_content_view_with_same_repos(self):
+        """Create a Composite Content View with content view having same yum repos.
+        Add filters on the content views, package count for composite should not
+        be changed.
+
+        :id: 957f3758-ca1e-4a1f-8e7d-171750e0eb87
+
+        :expectedresults: package count for composite should not be changed in case of mismatch.
+
+        :bz: 1639390
+
+        :CaseImportance: Medium
+        """
+        org = entities.Organization().create()
+        product = entities.Product(organization=org).create()
```

Comment:
Hey @omkarkhatavkar, I like the test and the helper function, though I suggest moving it to the ContentViewPublishPromoteTestCase class and then reusing the organization and the product from its setUpClass to save some time